### DISCUSSION TITLE
Aggregate all processes of the same program?

### DISCUSSION THREAD

**Discussion by ricardosalta** [winsiderss/systeminformer#1206] [Questions / Answers]
Specifically, I would like the option to condense the 77 processes Chrome currently has open on my machine, to get a definite understanding of how many resources it is actually using in total.

Extra: my own solution I just found, as I figured the way to count the number of processes to mention here, I just realized I can just (I just did in a short moment)

- pause the auto updating, 
- search for chrome.exe
- ctrl+a for select all
- ctrl+c for copy
- open Excel
- paste
- go to Data
- click Text to Columns
- run a sum() to the cpu column

I had to deal with the fact that by my regional settings, decimals are separated with a comma.
Since the text I got via ctrl+c was comma-separated, this caused a small head scratching.
No big issue though.

All-in-all I think I'm sorted for today with my own realisation, but for sure automating this sum thing would be great.

Anyway... thanks for your attention and possible ideas

**Comment by jxy-s** [comment]
In the nightly build there is an option to "Enable column header totals". With this enabled it shows a total at the top of the column. These totals are restricted to the filter. So for example, this is showing only the total for "firefox":

![image](https://user-images.githubusercontent.com/11687482/174415833-f32c7d91-b776-40e8-9f5b-39ebd13e254b.png)

Then checking how many resources certain processes are using is as simple as using the search box 😄. Notice the difference between the columns header totals and the status bar (at the bottom).
